osf/models/node.py

```diff
@@ -662,6 +662,14 @@ def csl(self):  # formats node information into CSL format for citation parsing
     def should_request_identifiers(self):
         return not self.all_tags.filter(name='qatest').exists()
 
+    @property
+    def meeting_submission(self):
+        """
+        First osfstoragefile on a node - if the node was created for a meeting,
+        assuming its first file is the meeting submission.
+        """
+        return self.files.first()
```

Comment:
Matt suggests finding a better place to stash this.  Would probably put on new MeetingsSubmissionSerializer directly then?